src/test/java/com/ddudu/following/service/FollowingServiceTest.java

```diff
@@ -0,0 +1,148 @@
+package com.ddudu.following.service;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+import com.ddudu.common.exception.DataNotFoundException;
+import com.ddudu.common.exception.DuplicateResourceException;
+import com.ddudu.following.domain.Following;
+import com.ddudu.following.dto.request.FollowRequest;
+import com.ddudu.following.dto.response.FollowResponse;
+import com.ddudu.following.exception.FollowingErrorCode;
+import com.ddudu.following.repository.FollowingRepository;
+import com.ddudu.user.domain.User;
+import com.ddudu.user.repository.UserRepository;
+import java.util.Optional;
+import net.datafaker.Faker;
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator.ReplaceUnderscores;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@Transactional
+@DisplayNameGeneration(ReplaceUnderscores.class)
+class FollowingServiceTest {
+
+  static final Faker faker = new Faker();
+
+  @Autowired
+  UserRepository userRepository;
+
+  @Autowired
+  FollowingService followingService;
+
+  @Autowired
+  FollowingRepository followingRepository;
+
+  @Autowired
+  PasswordEncoder passwordEncoder;
+
+  @Nested
+  class 팔로잉_생성_테스트 {
+
+    private User createUser() {
+      String email = faker.internet()
+          .emailAddress();
+      String password = faker.internet()
+          .password(8, 40, true, true, true);
+      String nickname = faker.oscarMovie()
+          .character();
+
+      User user = User.builder()
+          .passwordEncoder(passwordEncoder)
+          .email(email)
+          .password(password)
+          .nickname(nickname)
+          .build();
+
+      return userRepository.save(user);
+    }
+
+    @Test
+    void 팔로워가_존재하지_않으면_팔로잉_생성을_실패한다() {
```

Comment:
```suggestion
    void 사용자가_존재하지_않으면_팔로잉_생성을_실패한다() {
```

개인적으로 `팔로워가 존재하지 않으면` 이라는 문장이 살짝 어색해서 `사용자가 존재하지 않으면`은 어떠신가요? 음.. 다시 보니 또 어색한 것 같지는 않고.. 어떻게 생각하시나요?